west.yml

```diff
@@ -45,7 +45,8 @@ manifest:
   defaults:
     remote: ncs
 
-  group-filter: [-homekit, -nrf-802154, -dragoon, -find-my, -ant, -babblesim, -sidewalk, -bsec]
+  group-filter: [-homekit, -nrf-802154, -dragoon, -find-my, -ant, -babblesim, -sidewalk,
```

Comment:
seems like there is an issue with NordicBuilder script which we should investigate. I'll fix it manually for this PR